src/components/show-products/index.js

```diff
@@ -43,28 +51,56 @@ export const ShowProducts = () => {
 
   return (
     <ContentContainer>
-      <ul>
-        {products?.map(({ id, productName, lastPurchaseDate, timeFrame }) => (
-          <li className="checkbox-item" key={id}>
-            <label htmlFor={id} className="checkbox-label">
-              <input
-                type="checkbox"
-                id={id}
-                name={productName}
-                onChange={() => handleOnChange(id)}
-                checked={
-                  lastPurchaseDate && compareDates(lastPurchaseDate.toDate())
-                }
-                ariaLabel={timeFrames[timeFrame]}
-              />
-              <span
-                className={`checkmark checkbox-timeFrame-${timeFrame}`}
-              ></span>
-              <span className="checkbox-name">{productName}</span>
-            </label>
-          </li>
-        ))}
-      </ul>
+      <form className="filter-form">
+        <label htmlFor="filter">
+          Filter items
+          <input
+            className="search-item"
+            type="search"
+            id="filter"
+            name="filter"
+            aria-label="Search through list content."
+            placeholder="Start typing here..."
+            value={searchTerm}
+            onChange={(event) => setSearchTerm(event.target.value)}
+          />
+        </label>
+        {searchTerm && (
+          <input
+            class="close-icon"
```

Comment:
I believe it's a little better `className` than `class` in React. 👀 😄 

**Note.-** It gave me a small error in the browser console when I tested your PR in the local environment. 🤣

Reply:
You're right, Alejandro. We will fix this.